requirements.d/development.txt

```diff
@@ -6,23 +6,16 @@ py
 mock
 pycodestyle
 pytest
-pytest-cache
-pytest-pep8
-pytest-flake8
 pytest-xdist
 pytest-cov
 pytest-timeout
 sphinx
 sphinx_rtd_theme
 gitpython
-gevent
 pynco
 conda-verify
-Flask-Testing
 pytest-reverse
-eventlet>0.30.2
 dnspython>=2.0.0, <2.3.0
 gsl==2.7.0
 boa
 xmlschema<2.5.0
-
```

Comment:
We need any file that is not empty  to end with a new-line character, this is relevant for all POSIX systems and I think it comes from that historical decission http://gcc.gnu.org/ml/gcc/2003-11/msg01568.html

I can't  see hidden chars here, but we have to take care for this.

Reply:
This file is not missing a newline character at the end, but it previously had two. My editor automatically drops duplicate newlines at the end of file. GitHub shows a red sign if there is indeed a missing newline, as you can see e.g. here: https://github.com/Open-MSS/MSS/pull/2069/files/a2e10e1ce75daa73306a7e1076f124a3ad8501da#r1384630111 (this instance was fixed later).